src/common/layout/ContentPage.js

```diff
@@ -73,48 +74,49 @@ const FAQRoute = ({ history }: Props) => (
 );
 
 const ContentPage = (props: Props) => (
-  <SelectedBooking.Consumer>
-    {({ bookingPage, selectedBooking }) => (
-      <div className="ContentPage">
-        <Header />
-        <div className="Body">
-          {bookingPage === 'ALL_BOOKINGS' ? (
-            <div className="BookingInfo">
-              <BookingPage
-                bookingPage={bookingPage}
-                selectedBooking={selectedBooking}
-              />
-            </div>
-          ) : (
-            <Desktop>
-              <UserStatus.LoggedIn>
+  <BookingState.Consumer>
```

Comment:
this change - having booking available for the whole ContentPage - will be much appreciated 👍 💯